### ISSUE TITLE
stable redirect to codename for content page

### ISSUE THREAD

**Issue by 532910** [Debian/debiman#136]
While https://manpages.debian.org/stable/gdisk/gdisk.8.en.html works fine, https://manpages.debian.org/stable/gdisk/index.html does not redirect to https://manpages.debian.org/bullseye/gdisk/index.html

**Comment by stapelberg** [comment]
Yeah, redirects are configured explicitly, and the index pages are just not part of that database.

We probably could add the index pages, but not sure if/when I’ll get around to that.
